core/src/main/java/io/aiven/klaw/service/EmailService.java

```diff
@@ -107,7 +107,7 @@ public void sendSimpleMessage(
       message.setReplyTo(new Address[] {address});
       message.setFrom(new InternetAddress(fromMailId, noReplyMailIdDisplay));
 
-      text = text.replaceAll("\\\\n", "<br>");
+      text = text.replace("\n", "<br>");
```

Comment:
Sorry @snuyanzin this should have two slashes, I realise now that git removed one of the slashes I put in 😮‍💨 
`text = text.replace("\\n", "<br>");`

Reply:
thanks for checking
let's try with 2